game.cpp

```diff
@@ -246,9 +274,21 @@ bool Game::RunFrame(double dt) {
         renderer->ImDrawSphere(collisionInfo.hitPoint, 5.0f);
 #endif
 
-        //renderer->SetActiveCamera(&m_pFlyCameraEntity->m_Camera);
+        // FIX: Remove later. Ugly.
+        renderer->SetActiveCamera(renderCam);
         HKD_Model* playerColliderModel[] = { m_pPlayerEntity->GetModel() };
-        renderer->RenderColliders(renderCam, playerColliderModel, 1);
+        std::vector<BaseGameEntity*> pAllEntities = m_pEntityManager->Entities();
+        for (int i = 0; i < pAllEntities.size(); i++) {
+            BaseGameEntity* pEntity = pAllEntities[i];
+            if (pEntity->Type() == ET_ENEMY) {
+                Enemy* pEnemy = (Enemy*)pEntity;
+                HKD_Model* pModel = pEnemy->GetModel();
+                if (pModel == nullptr) {
+                    continue;
+                }
```

Comment:
is this check necessary? we already checked for the type. or do we want to have enemiew without a model

Reply:
Good question. In theory this could happen. But you're right. I am going to put an assert in here instead to show intent that we expect our enemies to have a model. This is throw away code anyways.